src/include/OSL/rendererservices.h

```diff
@@ -411,7 +411,8 @@ class OSLEXECPUBLIC RendererServices {
     virtual uint64_t register_string (const std::string& str,
```

Comment:
It looks like `register_string` is vestigial now. Does it make sense to remove it? Or maybe there is some potential use beyond a fancy wrapper for `ustring::hash`?

Reply:
I think you're right, but I wasn't sure about the compatibility break that would be incurred to removing a virtual method from the parent class. Maybe I'll just mark it as deprecated first?

Reply:
YOLO, removed it entirely.

Reply:
Oops, yeah, fixing.

Reply:
Other than this, did the overall simplification look sane to  you, @tgrant-nv ?

Reply:
Yes, definitely. It's a huge improvement in readability and maintainability for GPU strings. I don't think anyone will miss trying to reason about `load_device_string`.